eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/Relayer.scala

```diff
@@ -139,12 +139,13 @@ object Relayer extends Logging {
    */
   case class GetOutgoingChannels(enabledOnly: Boolean = true)
   case class OutgoingChannel(nextNodeId: PublicKey, channelUpdate: ChannelUpdate, commitments: AbstractCommitments) {
-    def toUsableBalance: UsableBalance = UsableBalance(
+    def toChannelBalance: ChannelBalance = ChannelBalance(
       remoteNodeId = nextNodeId,
       shortChannelId = channelUpdate.shortChannelId,
       canSend = commitments.availableBalanceForSend,
       canReceive = commitments.availableBalanceForReceive,
-      isPublic = commitments.announceChannel)
+      isPublic = commitments.announceChannel,
+      isEnabled = channelUpdate.channelFlags.isEnabled)
```

Comment:
Actually there is a subtlety here: `isEnabled` only applies to our side of the channel, there is a separate `channel_update` for the other direction (our peer -> us).

Ideally we would expose two bools: `sendEnabled` and `receiveEnabled` (our current `isEnabled` is actually `sendEnabled`) to have the most accurate data. However the relayer only keeps our own `channe_ update`, not our peer's `channel_update` (because we only want to know if we can send, we don't care about receiving).

I really don't think it's worth fixing, unless someone has a very compelling use-case for it, but I thought it was interesting to raise the point.